js/augurs-transforms-js/src/lib.rs

```diff
@@ -14,6 +14,8 @@ use augurs_forecaster::transforms::{StandardScaler, Transformer, YeoJohnson};
 #[serde(rename_all = "camelCase", tag = "type")]
 #[tsify(from_wasm_abi)]
 pub enum Transform {
+    /// Scale the data to a given range.
```

Comment:
```suggestion
    /// Scale the data to the range [0, 1].
```

Since we're not exposing the custom `output_range` it'll always be [0, 1].